roles/common/tasks/main.yml

```diff
@@ -12,6 +12,36 @@
   when: item.value.site_hosts | rejectattr('canonical', 'defined') | list | count
   tags: [letsencrypt, wordpress]
 
+- name: Verify dict format for apt package component variables
+  fail:
+    msg: "{{ lookup('template', 'package_vars_wrong_format_msg.j2') }}"
+  when: package_vars_wrong_format | count
+  vars:
+    package_vars:
+      apt_packages_default: "{{ apt_packages_default }}"
+      apt_packages_custom: "{{ apt_packages_custom }}"
+      memcached_packages_default: "{{ memcached_packages_default }}"
+      memcached_packages_custom: "{{ memcached_packages_custom }}"
+      php_extensions_default: "{{ php_extensions_default }}"
+      php_extensions_custom: "{{ php_extensions_custom }}"
+      sshd_packages_default: "{{ sshd_packages_default }}"
+      sshd_packages_custom: "{{ sshd_packages_custom }}"
+    package_vars_wrong_format: "[{% for k,v in package_vars.iteritems() if v | type_debug != 'dict' %}'{{ k }}',{% endfor %}]"
+  tags: [sshd, memcached, php]
+
+- name: Verify dict format for apt package combined variables
+  fail:
+    msg: "{{ package_vars_wrong_format_msg }}"
```

Comment:
this one should be `lookup('template'...` too?

Reply:
done